sound/soc/sof/ipc4-topology.c

```diff
@@ -813,11 +816,31 @@ static int sof_ipc4_widget_setup_comp_process(struct snd_sof_widget *swidget)
 		goto free_available_fmt;
 	}
 
+	switch (process->process_type) {
+	case SOF_PROCESS_KEYWORD_DETECT: 
+	{
+		struct sof_process_intelwov_cfg *cfg;
+
+		cfg = kzalloc(sizeof(struct sof_process_intelwov_cfg), GFP_KERNEL);
+		if (!cfg)
+			goto free_available_fmt;
+		process->ipc_config_data = cfg;
+		break;
+	}
+	case SOF_PROCESS_MICSEL:	/* fallback */
```

Comment:
fallback? fallthrough you mean? But why do we need to add these cases if you're doing nothing here. Just have a default

Reply:
OK. I will use the `default`.